### ISSUE TITLE
Installation instructions for using Graphviz and Tikz would be helpful

### ISSUE THREAD

**Issue by schlichtanders** [AlgebraicJulia/Catlab.jl#516]
I am on nixos and trying to use Graphviz and Tikz, however that is not straightforward.

It would be nice to have instructions about how to enable the two output formats. I guess it is because the dependencies are not wrapped as part of Julia, however it requires to find certain executables on the path.

Some information about how the process is officially intended would be helpful.

**Comment by epatters** [comment]
Right, Graphviz and TikZ use external dependencies, and this should be documented.

The Graphviz module will also use [Graphviz_jll](https://github.com/JuliaBinaryWrappers/Graphviz_jll.jl) if it is loaded, but that probably isn't documented either. Also, last time I checked, the Graphviz_jll was only partially functional, e.g., it could output text formats like JSON and SVG but not PNG or PDF.

**Comment by epatters** [comment]
Our resident Nix user @olynch could probably even provide on Nix-specific instructions.
